### DISCUSSION TITLE
Opening cyclotron lid for n-filter

### DISCUSSION THREAD

**Discussion by Yanks1023** [gpstar81/GPStar-proton-pack#98] [General]
So I'm nervous about opening the lid for the n filter led. I've seen videos of the lid not snapping back together properly once the glue bonds or tabs have been broken. Anyone have any problems or suggestions here? Two flatheads and go slow, I understand but getting it back together is my concern.

**Comment by gpstar81** [comment]
There is an alternative option.

In the update branch that i've been adding support for the PCB boards, I added another n-filter led which can go directly off pin #46. This is for a standard LED diode only, which would be receiving 5V. So you would need to add the appropriate resistor. Then you can stick the LED anywhere you wish.

**Reply by Yanks1023** [reply]
So if using this option, the led flashes on a overheat like the neopixel?

**Reply by gpstar81** [reply]
Yes, it is timed to flash and strobe at the same time as the NeoPixel option. Except you will not have colour changes, but only be the colour of the LED diode you choose.

**Reply by Yanks1023** [reply]
Sounds great. I didnt realize the nfilter neo changed colors. When does it change colors? When you use video game modes?

**Reply by gpstar81** [reply]
N-Filter flashes during continuous firing have different colours depending on which power mode the wand is in while in Proton mode. In video game modes, they are the video game colours for the current mode during this sequence.

During the ribbon cable alarm, it is red.

Overheating it is more blue/white.

**Reply by Yanks1023** [reply]
Also, is this supported in older code? For example - before you added the option for one or three less for 1984 mode. I haven't updated my code since then.

**Reply by gpstar81** [reply]
Only in the updates branch that i've been working on lately for adding support to the new PCB's. I have yet to test this code on the home built Arduino builds yet, but it should work. I plan to merge it into the main branch sometime this month.

**Reply by Yanks1023** [reply]
Okay, so if I do decide to open the lid, the neopixel is supported in the older code? The color changing neo is persuading my decision a little. Lol

**Reply by gpstar81** [reply]
Correct, the NeoPixel is supported in both and will continue to be supported. The LED option is coming in the update. This was added for the very reason as some people may be hesitant to pop open the cyclotron lid, so I added an easier option.

It is possible to use both at the same time in the coming update.

**Reply by Yanks1023** [reply]
Amazing as always!! Thank you! Maybe I can use superglue on the broken bonds to get it back together properly if I run into issues.

**Reply by gpstar81** [reply]
Just take your time and pop it off where it is indicated in the image in the spots to do so. Go very light, it doesn't need much pressure to break the bond/tabs. You will hear it when it happens.

I never glued mine back, it is sturdy enough after pressing it back together and is held by the n-filter part.

**Comment by Yanks1023** [comment]
I just realized the n filter led changes color during overheating depending on which video game mode your in. Amazing!! Great work!

**Reply by gpstar81** [reply]
Yup. Did you manage to get the latest code built and running on your diy system?

**Comment by Yanks1023** [comment]
Not yet. Have to put aside some time, but I’m getting ready to do so soon. On Jul 21, 2023, at 4:44 PM, Michael Rajotte ***@***.***> wrote:﻿
Yup. Did you manage to get the latest code built and running on your diy home build system?
